xpdacq/analysis.py

```diff
@@ -89,16 +90,14 @@ def _timestampstr(timestamp):
     #corrected_timestampstring = timestampstring.replace(':','-')
     return timestampstring
 
-def save_last_tif():
+def save_last_tiff():
```

Comment:
In general, if you are going to do rename like this you should probably include

```python
save_last_tif = save_last_tiff
```

to make sure you do not break any code on the beamline computers.